app/views/user_mailer/add_email.html.erb

```diff
@@ -0,0 +1,36 @@
+<p class="lead">
+  <%= t('user_mailer.add_email.header') + ' ' +
+    t('user_mailer.add_email.footer', confirmation_period: @confirmation_period) %>
```

Comment:
WDYT of two separate interpolations?

```suggestion
  <%= t('user_mailer.add_email.header') %>
  <%= t('user_mailer.add_email.footer', confirmation_period: @confirmation_period) %>
```

Reply:
Seems good to me. I'll double check that the content/layout isn't affected.